sycl/test/bit_cast/bit_cast.cpp

```diff
@@ -0,0 +1,91 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t1.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
+// RUN: %CPU_RUN_PLACEHOLDER %t1.out
+// RUN: %GPU_RUN_PLACEHOLDER %t1.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+//==---- bit_cast.cpp - SYCL bit_cast test -------------------==//
```

Comment:
There is no need in copyright headers for tests

Reply:
Thanks, removed.